packages/0x.js/src/index.ts

```diff
@@ -9,6 +9,7 @@ export {
     ERC20ProxyWrapper,
     ERC721ProxyWrapper,
     ForwarderWrapper,
+    OrderValidatorWrapper,
```

Comment:
I think we're now exposing types here as well, `OrderAndTraderInfo` etc.

Reply:
I see, yea thats the source of the `test-doc-generation` ci failure